CHANGELOG

```diff
@@ -1,10 +1,14 @@
-Changes to 0.4.29:
+Changes to 0.4.28:
```

Comment:
Why version got reduced? It might even make sense to bump major version

Reply:
0.4.28 is not yet released, correcting the typo here

Reply:
Thank you for the clarification, it's resolved. 

I lack permissions here to mark this conversation as resolved :)